.github/workflows/on-demand-ocr-soak-test.yml

```diff
@@ -73,6 +76,7 @@ jobs:
       OCR_CHAINLINK_NODE_FUNDING: ${{ inputs.chainlinkNodeFunding }}
       OCR_TIME_BETWEEN_ROUNDS: ${{ inputs.timeBetweenRounds }}
       TEST_LOG_LEVEL: debug
+      ENV_JOB_IMAGE: ${{ secrets.QA_AWS_ACCOUNT_NUMBER }}.dkr.ecr.${{ secrets.QA_AWS_REGION }}.amazonaws.com/chainlink-tests:${{ github.head_ref || github.ref_name }}
```

Comment:
Just curious why we aren't making this global like for the smoke tests?

Reply:
The smoke tests will reuse it across jobs once things like solana are updated whereas here it is all contained in the one job so it doesn't need to be global. It could honestly go either way and not make a difference.